### PR TITLE
Document `linear_stiffness` in SoftBody3D

### PR DESCRIPTION
**Pull request by SamDevelopsCode**

### CODE REVIEW

File: doc/classes/SoftBody3D.xml [hunk 1]
```diff
@@ -104,9 +104,10 @@
 		<member name="disable_mode" type="int" setter="set_disable_mode" getter="get_disable_mode" enum="SoftBody3D.DisableMode" default="0">
 			Defines the behavior in physics when [member Node.process_mode] is set to [constant Node.PROCESS_MODE_DISABLED]. See [enum DisableMode] for more details about the different modes.
 		</member>
-		<member name="drag_coefficient" type="float" setter="set_drag_coefficient" getter="get_drag_coefficient" default="0.0">
+		<member name="drag_coefficient" type="float" setter="set_drag_coefficient" getter="get_drag_coefficient" default="0.0">
```
Comment (AThousandShips) [inline doc/classes/SoftBody3D.xml:107]:
```suggestion
		<member name="drag_coefficient" type="float" setter="set_drag_coefficient" getter="get_drag_coefficient" default="0.0">
```

File: doc/classes/SoftBody3D.xml [hunk 2]
```diff
@@ -104,9 +104,10 @@
 		<member name="disable_mode" type="int" setter="set_disable_mode" getter="get_disable_mode" enum="SoftBody3D.DisableMode" default="0">
 			Defines the behavior in physics when [member Node.process_mode] is set to [constant Node.PROCESS_MODE_DISABLED]. See [enum DisableMode] for more details about the different modes.
 		</member>
-		<member name="drag_coefficient" type="float" setter="set_drag_coefficient" getter="get_drag_coefficient" default="0.0">
+		<member name="drag_coefficient" type="float" setter="set_drag_coefficient" getter="get_drag_coefficient" default="0.0">            
 		</member>
 		<member name="linear_stiffness" type="float" setter="set_linear_stiffness" getter="get_linear_stiffness" default="0.5">
+            Lower values will increase the SoftBody's ability to bend while higher values will make the Softbody more rigid.
```
Comment (AThousandShips) [inline doc/classes/SoftBody3D.xml:110]:
```suggestion
			Lower values will increase the SoftBody's ability to bend while higher values will make the Softbody more rigid.
```

**Review by AThousandShips — CHANGES_REQUESTED** [review]
Formatting needs fixing

**Review by AThousandShips — APPROVED** [review]

**Review by aaronfranke — APPROVED** [review]

File: doc/classes/SoftBody3D.xml [hunk 3]
```diff
@@ -107,6 +107,7 @@
 		<member name="drag_coefficient" type="float" setter="set_drag_coefficient" getter="get_drag_coefficient" default="0.0">
 		</member>
 		<member name="linear_stiffness" type="float" setter="set_linear_stiffness" getter="get_linear_stiffness" default="0.5">
+			Higher values will result in a more rigid SoftBody, while lower values will increase the SoftBody's ability to bend. On a range of 0 to 1.
```
[the remark this answers is not in the document]
Comment (aaronfranke) [reply doc/classes/SoftBody3D.xml:110]: "a more rigid body" might be confusing to read considering there is RigidBody, perhaps it should say "a more rigid soft body"?

[the remark this answers is not in the document]
Comment (akien-mga) [reply doc/classes/SoftBody3D.xml]: "will increase the rigidity of the body"?

**Review by akien-mga — APPROVED** [review]

**Comment by akien-mga** [comment]
Thanks! And congrats for your first merged Godot contribution :tada:

**Comment by SamDevelopsCode** [comment]
Thanks everyone for helping me with the issues and to grow as a developer. Love Godot!

**Comment by YuriSizov** [comment]
Cherry-picked for 4.1.3.

**Pull request merged**